src/site/content/en/blog/next-gen-css-2019/index.md

```diff
@@ -0,0 +1,318 @@
+    <source src="https://storage.googleapis.com/web-dev-assets/next-gen-css-2019/sticky-stack.mp4" type="video/mp4; codecs=h264">
+  </video>
+</figure>
+
+### [Sticky Slide](https://codepen.io/argyleink/pen/abbJOjP)
+
+Here, the sticky elements are cousins. (That is, their parents are siblings.) When a sticky element hits the lower boundary of its container, it moves up with the container, creating the impression that lower sticky elements are pushing up higher ones. In other words, they appear to compete for the stuck position.
+
+<figure class="w-figure w-figure--center">
+  <video controls autoplay loop muted>
+    <source src="https://storage.googleapis.com/web-dev-assets/next-gen-css-2019/sticky-slide.webm" type="video/webm; codecs=vp8">
+    <source src="https://storage.googleapis.com/web-dev-assets/next-gen-css-2019/sticky-slide.mp4" type="video/mp4; codecs=h264">
+  </video>
+</figure>
+
+### [Sticky Desperado](https://codepen.io/argyleink/pen/qBBrbyx)
+
+Like Sticky Slide, the sticky elements in this demo are cousins. However, they've been placed in containers set to a two-column grid layout.
+
+<figure class="w-figure w-figure--center">
+  <video controls autoplay loop muted>
+    <source src="https://storage.googleapis.com/web-dev-assets/next-gen-css-2019/sticky-desperado.webm" type="video/webm; codecs=vp8">
+    <source src="https://storage.googleapis.com/web-dev-assets/next-gen-css-2019/sticky-desperado.mp4" type="video/mp4; codecs=h264">
+  </video>
+</figure>
+
+## `backdrop-filter`
+
+The [`backdrop-filter`](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter) property lets you apply graphical effects to the area _behind_ an element rather than to the element itself. This makes lots of cool effects that were previously only achievable using complicated CSS and JavaScript hacks doable with one line of CSS.
+
+For example, this demo uses `backdrop-filter` to achieve native OS-style blurring:
+
+<iframe height="510" style="width: 100%;" scrolling="no" title="mddjjor" src="https://codepen.io/una/embed/mddjjor?height=265&theme-id=dark&default-tab=result" frameborder="no" allowtransparency="true" allowfullscreen="true">
+  See the Pen <a href='https://codepen.io/una/pen/mddjjor'>mddjjor</a> by Una Kravets
+  (<a href='https://codepen.io/una'>@una</a>) on <a href='https://codepen.io'>CodePen</a>.
+</iframe>
+
+We already have a [great post about `backdrop-filter`](https://web.dev/backdrop-filter/), so head there for more info.
+
+## `:is()`
+
+While the [`:is()` pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) is actually over ten years old, it still doesn't see as much use as we think it deserves. It takes a comma-separated list of selectors as its argument and matches any selectors in that list. That flexibility makes it incredibly handy and can significantly reduce the amount of CSS you ship.
+
+Here's a quick example:
```

Comment:
missing code sample? i do like the idea of having code in a video and then fenced accessible code for copy/paste.

Reply:
I took it out because I felt having duplicative figures isn't ideal because it implies they're conveying different information. But I'll put it back in.